runtime/manual/basics/modules/grpc_connections.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "gRPC Connections"
+---
+
+gRPC is a high-performance, open-source, universal RPC framework that enables
+efficient communication between services. With gRPC support, you can build
```

Comment:
What prompted this one? GRPC is hardly in the "basics" field and I think we shouldn't mention it so prominently in the manual 😅

Reply:
it was in the 1.4 release blog and didn't have documentation, so thought I'd add it in. Is there a better place for it to go?

Reply:
I think this at least belongs in a different PR but I agree with Bartek that maybe not in the manual at all. The support Deno provides is better node compatibility to run gRPC packages - it doesn’t have direct gRPC support.

Reply:
I'll remove it